src/BloomBrowserUI/bloomUI.less

```diff
@@ -209,3 +209,7 @@ body.bloom-hidden-for-animation > *:not(div.bloom-animationPreview) {
 body.bloom-hidden-for-animation {
     overflow: hidden; //hide the scrollbar during an animation preview.
 }
+
+.bloom-link-grid {
+    min-height: 100px;
```

Comment:
Even with the current FAL book grid, if you delete all the books in it, its size goes to nothing so you can't click on it. Want to prevent that somehow but not sure exactly how